src/db/manifest.rs

```diff
@@ -31,7 +33,10 @@ impl Manifest {
             .entry(name.clone())
             .or_insert_with(|| LogRegistrant {
                 name,
-                created_at: SystemTime::now(),
+                created_at: SystemTime::now()
+                    .duration_since(SystemTime::UNIX_EPOCH)
```

Comment:
2038 problem. Thats 18 years so prob dont need to worry about it right now.

Reply:
I'm in transit so I can't check, but I think it returns an unsigned 32bit int, which would make it a 2106 problem. If it's a signed 32bit, then we should just make it an explicit u64 and be done with it.  At that point it's year 584,554,051,223's problem.